scalableminds/webknossos
# Allow Superusers to Join Organizations Unlisted

For webKnossos maintenance, superUsers can now join organizations without being listed as a user there

**NOTE** that #5149 should be merged first for consecutive schema numbering

### Steps to test:
- create multi-organization setup (e.g. via isDemoOrganization = true)
- create a second organization
- as default user (who is a superuser), switch to the new organization, should not be listed there
- when joining an organization as a normal user, you should be listed there

### Issues:
- fixes #5090 
------
